script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -522,27 +528,27 @@ The smile on her face made me apologize by reflex.
 [sha:6d9f6b5732a76b39b920c42eab363c62b7687ed8]{
 -- Page 37, Offset 1247. Mods: @k, @e.
 -- 「いえいえ、目立つようなコトをしたわたしに非があるんです。
-"No, it's my fault for doing something so conspicuous.
+"No, it's my fault for standing out so much.
 }
 [sha:75f413ee875552c28b95b473048099cdb5ebb1f8]{
 -- Page 37, Offset 1248.
 -- 先輩として恥ずかしいところを見せてしまいました。
-I accidentally let you see an embarrassing side of your upperclassman.
+I'm sorry you had to see your upperclassman like that.
 }
 [sha:b6427c01fa14beeabf2c10215fae6ddc043ca355]{
 -- Page 37, Offset 1249.
 -- でも、朝は弱いって話、ホントなんですね」
-I guess that saying about not being in peak condition in the mornings really is true, huh?"
+Still, it seems they were right about you not being a morning person."
```

Comment:
This actually makes total sense, fits perfectly with the reply later on. Ciel manipulation starting early.

Just a question for my own selfish Japanese learning, but is it just from context that you can extrapolate she's talking about Shiki and not herself? Or is there something within the phrasing that suggests this.

Reply:
I dunno, she kinda changes the subject, so it felt like the only way to interpret it. I can't really explain it well. It doesn't really make sense for her to talk generally here in any way.